VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -1123,7 +1123,7 @@ export abstract class WaitingRoomBaseDirective {
     }
 
     shouldCurrentUserJoinHearing(): boolean {
-        return !this.isHost() || this.dualHostHasSignalledToJoinHearing;
+        return this.isHost() ? this.dualHostHasSignalledToJoinHearing || this.participant.status === ParticipantStatus.InHearing : true;
```

Comment:
have you tried the ejud panel member in the hearing, beacuse Ejud panel members have role Judge

Reply:
No but that would then give them access to the judge view right?
Is this expected? I will have a look now

Reply:
Checked this and the isHost works correctly for ejud

![image](https://user-images.githubusercontent.com/8798036/139708311-d82626bc-cccb-4932-9584-d44dba3179ff.png)